automate.py

```diff
@@ -0,0 +1,113 @@
+
+"""" script to generate image caption, alt-text and image-directive. """
+
+import os
+import shutil
+
+
+# move files from one directory to another
+# if files alrady exist there, they will be overwritten
+# retains original file date/time
+
+
+# source directory - generated screenshots are stored here
+
+dir_src = os.path.expanduser("~/collect/fastlane/metadata/android/en/images/phoneScreenshots/")
+
+# destination directory
+
+dir_dst = os.path.expanduser("~/odk/odkenv/docs/img/form-widgets")
+
```

Comment:
these file paths concern me, because they make assumptions about how the user has their files organized. (This is bad practice generally, BTW. For example -- this isn't how *I* have my files organized.)

The should be relative to the directory the user is in when they run the script, which should should be the root of the odk-docs repo.